epub34/rs/index.html

```diff
@@ -1089,7 +1088,7 @@ <h3>MathML</h3>
 							</li>
 						</ul>
 
-						<p class="note">Reading systems may choose to use third-party libraries such as <a
+						<p class="note">Reading systems can opt to use third-party libraries such as <a
```

Comment:
```suggestion
						<p class="note">Reading systems can opt for third-party libraries such as <a
```

May be wrong, but this sounds better to my ears...